src/DependencyInjection/Configuration.php

```diff
@@ -15,12 +14,9 @@
  */
 final class Configuration implements ConfigurationInterface
 {
-    /**
-     * {@inheritdoc}
-     */
     public function getConfigTreeBuilder(): TreeBuilder
     {
-        if (method_exists(TreeBuilder::class, 'getRootNode')) {
+        if (\method_exists(TreeBuilder::class, 'getRootNode')) {
```

Comment:
`getRootNode` exists in both 5.4 and 7.2, if this is a backwards compatibility check maybe something else might be needed. Or maybe this check if-else check can be removed because the `root` method is only available in Symfony 4.

https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Config/Definition/Builder/TreeBuilder.php
https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Config/Definition/Builder/TreeBuilder.php
https://github.com/symfony/symfony/blob/7.2/src/Symfony/Component/Config/Definition/Builder/TreeBuilder.php